### DISCUSSION TITLE
[Feature] I want set the button status to "active" when the cursor is over bold content. How to obtain this status

### DISCUSSION THREAD

**Discussion by nroyliu** [Milkdown/milkdown#1199] [Q&A]
### Initial checklist

- [x] I agree to follow the [code of conduct](https://github.com/Saul-Mirone/milkdown/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched [issues](hhttps://github.com/Saul-Mirone/milkdown/issues) and [discussions](https://github.com/Saul-Mirone/milkdown/discussions) and couldn’t find anything (or linked relevant results below)

### Problem

I want set the button status to "active" when the cursor is over bold content. How to obtain this status

### Solution

not

### Alternatives

not

**Comment by Saul-Mirone** [comment]
You can use `doc.rangeHasMark` to do that. Like how I did it here: https://github.com/Milkdown/milkdown/blob/main/e2e/src/link-tooltip/main.ts#L42
